### ISSUE TITLE
using old model results as priors for new model?

### ISSUE THREAD

**Issue by lorenze3** [google/lightweight_mmm#143]
Hi, I'd love a utility to use a fitted model to generate priors for an update to that same model.  Ideally there would be a way to generate the priors for new features (e.g. new media channel can be mapped to a previous media channel to be used as a prior), but at a minimum it would be awesome to be able to chain together the estimation process.  

I didn't immediately a way to specify a prior in numpyro that isn't a distribution, so I'm not sure this is possible . . . but it would be very useful.

**Comment by steven-struglia** [comment]
Agreed that some sort of `warm_start` parameter in the model would be great

**Comment by michevan** [comment]
Thanks a lot for the suggestion! I just wanted to share a link to our new "[support](https://github.com/google/lightweight_mmm#support)" section in the readme to give you some more information on how we can and cannot support such requests at the moment.
